convert2rhel/unit_tests/pkghandler_test.py

```diff
@@ -19,6 +19,7 @@
 import os
 import re
 import yum
+import sys
```

Comment:
Alphabetical import order
```suggestion
import sys
import yum
```